backend/src/app/rpc/commands/binfile.clj

```diff
@@ -336,11 +336,23 @@
       (->> (db/exec! conn [sql ids])
            (mapv #(assoc % :file-id id))))))
 
+(defn- get-file-tagged-object-thumbnails
+  [{:keys [::db/pool]} id]
+  (pu/with-open [conn (db/open pool)]
+    (let [sql (str "SELECT * FROM file_tagged_object_thumbnail WHERE file_id = ?")]
```

Comment:
You don't need call `str` here, if you only have one string segment and no need concat nothing